samples/grids/grid/editing-excel-style/src/index.tsx

```diff
@@ -1,146 +1,138 @@
-import React from 'react';
+import React, { useRef, useEffect } from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 
-import { IgrGridModule } from 'igniteui-react-grids';
+import { IgrActiveNodeChangeEventArgs, IgrGridModule } from 'igniteui-react-grids';
 import { IgrGrid, IgrColumn } from 'igniteui-react-grids';
-import { ComponentRenderer, WebGridDescriptionModule } from 'igniteui-react-core';
 import NwindData from './NwindData.json';
-import { IgrGridKeydownEventArgs } from 'igniteui-react-grids';
 
 import 'igniteui-react-grids/grids/combined';
 import 'igniteui-react-grids/grids/themes/light/bootstrap.css';
 
-const mods: any[] = [
-    IgrGridModule
-];
-mods.forEach((m) => m.register());
+// Register modules once
+IgrGridModule.register();
```

Comment:
This is no longer needed and can be removed as visible if ran with the 19 package:
![image](https://github.com/user-attachments/assets/683e9131-655b-4481-9cda-00fc6b3ead53)

Reply:
removed